test/EasyRdf/GraphTest.php

```diff
@@ -159,6 +159,13 @@ public function testParseFile()
         $this->assertSame(null, $name->getDatatype());
     }
 
+	public function testParseLargeFile()
```

Comment:
Can you remove the tabs? It prevents Travis check.
```suggestion
    public function testParseLargeFile()
    {
        $graph = new Graph();
        $count = $graph->parseFile(fixturePath('stw.rdf'));
        $this->assertSame(109340, $count);
    }
```